lib/domain/usecase/send_file_interactor.dart

```diff
@@ -0,0 +1,31 @@
+
+import 'package:fluffychat/presentation/extensions/room_extension.dart';
+import 'package:matrix/matrix.dart';
+
+class SendFileInteractor {
+  Future<void> execute({
+    required Room room,
+    required List<MatrixFile> matrixFiles,
```

Comment:
the same question for image. we have to transform `file` to `matrixfile`, how to do it inside interactor?